src/screens/DiscoveryScreenV2/elements/Search.js

```diff
@@ -180,7 +181,7 @@ const DiscoverySearch = ({
             ref={discoverySearchBarRef}
             testID={TestIdConstant.discoveryScreenSearchBar}
             focusable={true}
-            autoFocus={true}
+            autoFocus={autoFocus}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `autoFocus` prop is now being passed to the `TextInput` component. This will cause the input field to automatically gain focus when the component mounts.
```diff
- autoFocus={true}
+ autoFocus={autoFocus}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->